actionview/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add `config.action_view.nest_html_attributes_within`
```

Comment:
I struggled to name this configuration key. Please share any feedback and ideas you might have!

Reply:
Not better, but here's an alternative: `allow_nested_html_attributes_for`

I'm worried this sounds too similar `accept_nested_attributes`